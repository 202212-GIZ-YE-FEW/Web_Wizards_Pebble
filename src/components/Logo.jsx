import Image from "next/image";

import logo from "~/logo.svg";

export default function Logo(props) {
    const { className } = props;
    return (
        <Image
            src={logo}
            alt='PebbleWork Logo'
            priority
            className={`h-[32.32px] w-p[100.75px] md:h-[69.13px] md:w-[215.44px] ${className}`}
        />
    );
}
