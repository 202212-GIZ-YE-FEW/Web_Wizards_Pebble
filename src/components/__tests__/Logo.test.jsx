import renderer from "react-test-renderer";

import Logo from "@/components/Logo";

it("renders correctly", () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
});
