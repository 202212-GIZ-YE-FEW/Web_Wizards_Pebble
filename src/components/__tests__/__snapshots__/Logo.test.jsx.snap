// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="PebbleWork Logo"
  className="h-[32.32px] w-p[100.75px] md:h-[69.13px] md:w-[215.44px] undefined"
  data-nimg="1"
  decoding="async"
  height={40}
  onError={[Function]}
  onLoad={[Function]}
  src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
  srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
  style={
    Object {
      "color": "transparent",
    }
  }
  width={40}
/>
`;
